app/package.json

```diff
@@ -125,7 +136,12 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "./client/**/*.{js,jsx,ts,tsx}": [
+      "npx prettier --write"
+    ]
```

Comment:
add "eslint ./client --ext '.js, .jsx' " when ready...right now there is a whole load of issues using the airbnb react standard